### PR TITLE
Exploring what using JSON RPC may look like

### PR DESCRIPTION
**Pull request by andrewboudreau**
This is very much a work in progress still but wanted to solicit feedback and explore our options for an JSONRPC based solution.

### CODE REVIEW

File: miner_exporter.py [hunk 1]
```diff
@@ -74,120 +77,43 @@
...
+    log.error(f"docker failed while bootstrapping. Not exporting anything. Error: {ex}")
+    return  
+  
   # collect total cpu and memory usage. Might want to consider just the docker
   # container with something like cadvisor instead
-  SYSTEM_USAGE.labels('CPU', hotspot_name_str).set(psutil.cpu_percent())
-  SYSTEM_USAGE.labels('Memory', hotspot_name_str).set(psutil.virtual_memory()[2])
-  SYSTEM_USAGE.labels('CPU-Steal', hotspot_name_str).set(psutil.cpu_times_percent().steal)
-  SYSTEM_USAGE.labels('Disk Used', hotspot_name_str).set(float(psutil.disk_usage('/').used) / float(psutil.disk_usage('/').total))
-  SYSTEM_USAGE.labels('Disk Free', hotspot_name_str).set(float(psutil.disk_usage('/').free) / float(psutil.disk_usage('/').total))
```
Comment (andrewboudreau) [inline miner_exporter.py:177]: Opps, looks like some of these got deleted by mistake

**Comment by tedder** [comment]
thanks! I'll take this and @kellybyrd's stuff in #16, and maybe some things from #23, and start a redesign. Feel free to keep making changes here, no need to rebase because I'll just grab bits and pieces.

**Review by andrewboudreau — COMMENTED** [review]
> Feel free to keep making changes here, no need to rebase because I'll just grab bits and pieces.

Perfect, thanks

**Comment by andrewboudreau** [comment]
This pr in the miner, https://github.com/helium/miner/pull/909 will help hbbft penalties query

**Pull request closed without merging**